transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -321,14 +336,6 @@ int netty_unix_socket_getOption(JNIEnv* env, jint fd, int level, int optname, vo
     return rc;
 }
 
-int netty_unix_socket_setOption(JNIEnv* env, jint fd, int level, int optname, const void* optval, socklen_t len) {
```

Comment:
IIUC this method doesn't have to move (it isn't static and is declared in the header file)?